README.md

```diff
@@ -1,10 +1,24 @@
 # NodeSecure i18n
```

Comment:
Can you remove the title here please

Reply:
It's done.
sorry, I didn't see this little title :)